v21.1/backup.md

```diff
@@ -23,42 +23,52 @@ You can also back up:
+    `BACKUP` only offers table-level granularity; it _does not_ support backing up subsets of a table.
 
-{{site.data.alerts.callout_info}}
-`BACKUP` only offers table-level granularity; it _does not_ support backing up subsets of a table.
-{{site.data.alerts.end}}
+Because CockroachDB is designed with high fault tolerance, these backups are designed primarily for disaster recovery (i.e., if your cluster loses a majority of its nodes) through [`RESTORE`](restore.html). Isolated issues (such as small-scale node outages) do not require any intervention.
 
 {{site.data.alerts.callout_success}}
 To view the contents of an enterprise backup created with the `BACKUP` statement, use [`SHOW BACKUP`](show-backup.html).
 {{site.data.alerts.end}}
 
 ## Required privileges
 
-- Only members of the `admin` role can run `BACKUP`. By default, the `root` user belongs to the `admin` role.
+- Full cluster backups can only be run by members of the `admin` role. By default, the `root` user belongs to the `admin` role.
+- For all other backups, the user must have read access (`SELECT` or `USAGE`) on all objects being backed up.
 - `BACKUP` requires full read and write (including delete and overwrite) permissions to its target destination.
 
+### Destination privileges
+
+{% include {{ page.version.version }}/backups/destination-file-privileges.md %}
+
 ## Synopsis
 
 <div>
 {% include {{ page.version.version }}/sql/generated/diagrams/backup.html %}
 </div>
 
+{{site.data.alerts.callout_info}}
+<span class="version-tag">New in v21.1:</span> `BACKUP ... INTO` adds a backup to a collection within the backup destination. The path to the backup is created using a date-based naming scheme. Previous versions of CockroachDB used the syntax `BACKUP ... TO` to back up directly to a specific operator-chosen destination, rather than picking a date-based path. For more information on this old syntax, [see the docs for v20.2](../v20.2/backup.html) or earlier.
+{{site.data.alerts.end}}
+
 ## Parameters
 
  Parameter | Description
-------------|-------------
-`table_pattern` | The table(s) or [view(s)](views.html) you want to back up.
-`database_name` | The name of the database(s) you want to back up (i.e., create backups of all tables and views in the database).|
+-----------+-------------
+`targets` | Back up the listed [targets](#targets).
+`subdirectory` | The name of the specific subdirectory (e.g., `2021/03/23-213101.37`) where you want to add an incremental backup. To view available subdirectories, use [`SHOW BACKUPS IN destination`](show-backup.html). If the `subdirectory` is not provided, a full backup will be created in the collection using a date-based naming scheme (i.e., `<year>/<month>/<day>-<timestamp>`).<br><br>**Warning:** If you use an arbitrary `STRING` as the subdirectory, a new full backup will be created, but it will never be shown in `SHOW BACKUPS IN`. We do not recommend using arbitrary strings as subdirectory names.
```

Comment:
~Can "incremental backup" and "full backup" in the above be links to those sections?